OpenSim/Common/DataTable.h

```diff
@@ -311,6 +344,11 @@ class DataTable_ : public AbstractDataTable {
         return _depData.col((int)index);
     }
 
+    /** Get dependent Column which has the given column label.                */
+    VectorView getDependentColumn(const std::string& columnLabel) {
+        return _depData.col(getColumnIndex(columnLabel));
+    }
+
```

Comment:
Nice. Is there similar convenience access to the independent column data? For TimeSeriesTable you could also add `getTimes()` or `getTimeColumn()` which returns the independent column.